### DISCUSSION TITLE
I would like to discuss an easier way of inviting people into an instance using VRCX

### DISCUSSION THREAD

**Discussion by SapphireMedal1** [vrcx-team/VRCX#225] [Ideas]
Usually when I want to invite a friends list of 80 people to fill up a world, I have to individually click every account username and scroll down. This takes too much time and would like to talk about maybe adding an option where you could hold shift and click a certain portion of the list, and shift click the ending part of the list to add everything selected into a queue. 

Is there a way to add a list of favorite friends or similar all at once into the invite queue? This seems like such a basic feature that I struggle to see how it would be difficult to implement.

**Comment by Myrkie** [comment]
it would be cool if vrcx were to recongize the current instance you are in and add an option to invite everyone thats on your friends list that is in the curent instance to another world for a world swap were a portal is not a option

**Reply by vmfunc** [reply]
> 
> 
> it would be cool if vrcx were to recongize the current instance you are in and add an option to invite everyone thats on your friends list that is in the curent instance to another world for a world swap were a portal is not a option

Inviting everyone would be considered as botting.

**Comment by SapphireMedal** [comment]
I'm not asking for an invite everyone button. I'm asking for the option to be able to select-click and scroll to the amount of people you want to invite rather than having to individually click every single person you want to add to a world. Botting or not you are personally selecting individual people and you can spend 17 minutes selecting your entire friendslist and VRCX will force-push the invite list out to every single person on the list. How is that any different than having the option to multi-select people on the application? VRCX wont be banned, your account will if you abuse it, so there is no reason why this shouldn't be an option when its so easily implemented with fairly basic code.
